packages/plugin-list/test/Checklist.test.ts

```diff
@@ -223,7 +221,8 @@ describePlugin(List, testEditor => {
             });
             it('should parse a complet checklist', async () => {
                 await testEditor(BasicEditor, {
-                    contentBefore: unformat(`
+                    contentBefore: unformat(
+                        `
```

Comment:
If you care about that kind of things then why not `'<ul` rather than using an entire line for `'` ? I mean, the variable ends with `</ul>'` already so...